lib/daru/category.rb

```diff
@@ -64,6 +62,17 @@ def name= new_name
       self
     end
 
+    def plotting_library= lib
+      case lib
+      when :gruff
+        @plotting_library = lib
+        extend Daru::Plotting_Gruff::Category if Daru.has_gruff?
```

Comment:
I assume, because in some contexts there are conflicts with nyaplot's own namespace (and it should be accessed as `::Nyaplot` to show we want top-level module, not `Daru::Plotting::Category::Nyaplot`)